cmd/manager/main.go

```diff
@@ -842,7 +838,7 @@ func setupWebhook(
 	}
 
 	// esv1 validating webhook is wired up differently, in order to access the k8s client
-	esvalidation.RegisterWebhook(mgr, validateStorageClass, exposedNodeLabels)
+	esvalidation.RegisterWebhook(mgr, validateStorageClass, exposedNodeLabels, managedNamespaces)
```

Comment:
We need to cover the other web hooks as well that are not wired up like the Elasticsearch webhook but are using the standard controller-runtime webhook builder. Otherwise the web hooks will still validate resources in namespaces they are not responsible for.

Reply:
@pebrc this took some work, and a new common/webooks package to accomplish, but this is now handled in a consistent fashion across all non-elasticsearch webhooks (ES is special, although with a bit of work, we probably could make it fit this new 'mold' for webhooks, let me know if you want me to go down that path)